lighthouse-core/config/default.js

```diff
@@ -27,6 +27,8 @@ module.exports = {
   {
     "passName": "redirectPass",
     "useThrottling": false,
+    // Speed up the redirect pass by blocking stylesheets, fonts, and images
+    "blockedUrlPatterns": ["*.css", "*.jpg", "*.jpeg", "*.png", "*.gif", "*.svg", "*.ttf", "*.woff", "*.woff2"],
```

Comment:
not sure about this one, but does it make sense to have these apply across `beforePass`, `pass`, and `afterPass`? Maybe it would make more sense/be less surprising to have them blocked only during `pass`, analogous to `recordTrace`, `useThrottling`, etc cc/ @paulirish